### PR TITLE
feat(elm): beautify rooms view

### PR DESCRIPTION
**Pull request by jtkpiotr**
**Summary**
Display available rooms in Rooms Panel as tiles.

screen shot 2017-08-04 at 18 29 45

**Related issues**
Closes: #32

**Test plan**

### CODE REVIEW

File: aion/web/elm/src/General/View.elm [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+import Html.Attributes exposing (href, style, class)
 import Msgs exposing (Msg)
 import Routing exposing (panelPath, roomsPath)
 import RemoteData exposing (WebData)
-import Room.Models exposing (RoomsData)
+import Room.Models exposing (RoomsData, Room)
+import Bootstrap.Grid as Grid
+import Bootstrap.Grid.Col as Col
+import Bootstrap.CDN as CDN
+import Array
```
Comment (pmrukot) [inline aion/web/elm/src/General/View.elm]: I would reorder imports in alphabetical way, elm-format does that automatically:
https://github.com/avh4/elm-format/blob/master/Style%20Guide/Imports.md

**Review by mrapacz — APPROVED** [review]
Looks okay, not sure about the slicing part, but will take care of that while refactoring elm I guess. So far so good.

**Review by pmrukot — APPROVED** [review]

**Comment by pmrukot** [comment]
This issue came out:
https://travis-ci.org/pmrukot/aion/builds/264667775

So I locked the plug version as suggested here:
https://elixirforum.com/t/plug-mime-type-1-is-deprecated-please-use-mime-type-1-instead/7139

TL;DR -> Phoenix not ready for new Plug
